### DISCUSSION TITLE
Authentication for WebUI

### DISCUSSION THREAD

**Discussion by snaike00** [mitmproxy/mitmproxy#6880] [General]
Hello.
I have question - how to configure **authentication** (login/password) on mitmweb v10.3.0 **for Web Interface only**?
I was found this old thread https://github.com/mitmproxy/mitmproxy/pull/3496 but this solution not working.

**Comment by mhils** [comment]
We don't have builtin web interface authentication at the moment. Pull requests are welcome! :)
